README.md

```diff
@@ -34,3 +36,17 @@ Underneath the covers, we hit the DevOps Service ``https://console.bluemix.net/d
 3. Workspace URL, for example: ``chatbotWorkspaceURL=chatbotWorkspaceURL=https%3A%2F%2Fgithub.com%2FIBM%2Fchatbot-deployer%2Fraw%2Fmaster%2Fdata%2Fworkspace.json``
 
 Click [this link](https://console.bluemix.net/devops/setup/deploy?repository=https%3A%2F%2Fgithub.com%2FIBM%2Fchatbot-deployer&chatbotName=Chatbot&chatbotWorkspaceURL=https%3A%2F%2Fgithub.com%2FIBM%2Fchatbot-deployer%2Fraw%2Fmaster%2Fdata%2Fworkspace.json) to try it out.
+
+## Privacy Notice
+
+Sample DevOps pipeline that include this package may be configured to track deployments to [IBM Bluemix](https://www.bluemix.net/) and other platforms. The following information is sent to a [Deployment Tracker](https://github.com/IBM/metrics-collector-service) service on each deployment:
```

Comment:
I'd rephrase this as:

"The DevOps pipeline that is included in this package is configured by default with a [Deployment Tracker](https://github.com/IBM/metrics-collector-service) tool. The following information is sent when a user deploys an asset from the Bot Asset Exchange:"